lib/cli/cli.js

```diff
@@ -1,16 +1,17 @@
-'use strict';
+"use strict";
```

Comment:
I would prefer to keep styling refactors out of this PR.

Reply:
I'm fine with that. For some weird reason I was getting errors in the job because of the formatting, so I ran 'prettier' and ended up with this. I'll see about backing that out and starting over.